packages/server/utils/OpenAIServerManager.ts

```diff
@@ -20,20 +20,28 @@ class OpenAIServerManager {
     if (!this.openAIApi) return null
     try {
       const location = summaryLocation ?? 'retro meeting'
-      const response = await this.openAIApi.createCompletion({
-        model: 'text-davinci-003',
-        prompt: `Below is a comma-separated list of text from a ${location}. Summarize the text for a second-grade student in one or two sentences.
+      const response = await this.openAIApi.createChatCompletion({
+        model: 'gpt-3.5-turbo',
+        messages: [
+          {
+            role: 'user',
+            content: `Below is a comma-separated list of text from a ${location}.
+            Summarize the text for a second-grade student in one or two sentences.
+            If you can't provide a summary, simply say the word "No".
```

Comment:
I experimented a couple of variations of prompts. Turns out it needs more changes to output an ideal summary for the test case you provided (sth like "the meeting talks about target word"). To make this change small and minimize the impact on more real-word & meaningful use cases, I chose this approach to instruct ChatGPT says no if it's incapable of doing the job. WDYT? Safe to try?

Reply:
Yes, safe to try!